src/System.Linq.Expressions/tests/ExceptionHandling/ExceptionHandlingExpressions.cs

```diff
@@ -288,93 +282,60 @@ public void NonExceptionDerivedExceptionWrappedInterpreted()
             Assert.Equal("System.Runtime.CompilerServices.RuntimeWrappedException", ex.GetType().FullName);
```

Comment:
@terrajobst, @weshaggard, it looks like we don't expose RuntimeWrappedException in any of the ref assemblies.  Is that by design?  Is this not supported in .NET Core?